### DISCUSSION TITLE
Exception: Could not find field converter for column id (<class 'sqlmodel.sql.sqltypes.GUID'>)

### DISCUSSION THREAD

**Discussion by Anton-Karpenko** [aminalaee/sqladmin#155] [Q&A]
Hey, I am using sqlmodel to create models. I use the UUID type for the id columns.  

```
class RandomModel(SQLModel, table=True):
    id: uuid.UUID = Field(primary_key=True, index=True, nullable=False, default_factory=uuid.uuid4)
```

I added sqladmin to my project and I would like to create an instance within the admin panel. I cannot open `create` page because of an error.
`Exception: Could not find field converter for column id (<class 'sqlmodel.sql.sqltypes.GUID'>)`
Can I apply a custom converter to it?

**Comment by aminalaee** [comment]
Hey, thanks a lot for pointing this out.
I'd missed this,.
